upsert/upsert.go

```diff
@@ -0,0 +1,46 @@
+package upsert
+
+import (
+	"errors"
+	"net/http"
+	"strings"
+)
+
+var (
+	//ErrInvalidResponse is returned when the Search API returns a non 200 status
+	ErrInvalidResponse = errors.New("invalid status returned by search.api.ch.gov.uk")
+)
+
+// Interface for calling api to upsert data to an elastic search index
```

Comment:
Comment must begin with interface name
```suggestion
// Upsert is an interface for calling API to upsert data to an elastic search index
```